backend/routes.ts

```diff
@@ -63,7 +63,8 @@ app.post('/api/game', (req, res) => {
 
     GAMES.push(gameId);
     res.status(201).json({
-        id: gameId.id
+        id: gameId.id,
+        length: gameId.length
```

Comment:
This is a bit confusing to me, but I understand it better after reading the surrounding context. The problem is that the variable is called `gameId` which to me indicates that it's the ID of a game, when in fact it's an object that contains lots of information about a game, not just the ID.

I would just have called it `game` or `gameObject`.